tests/featurizers/test_featurizers.py

```diff
@@ -32,7 +32,7 @@ def _assert_featurized_patients_structure(labeled_patients, featurized_patients,
         labels_per_patient
     ), f"len(featurized_patients[1]) = {len(featurized_patients[1])}"
 
-    patient_ids = np.array(sorted([i for i in range(len(labeled_patients))] * len(labels_per_patient)))
+    patient_ids = np.array(sorted([i for i in labeled_patients] * len(labels_per_patient)))
```

Comment:
```suggestion
    patient_ids = np.array(sorted(list(labeled_patients) * len(labels_per_patient)))
```

Are these equivalent? I don't love the suggestion since still feels kind of weird - is this an API thing or something else (like just how the test is written)?